app/services/reject_by_default_feedback.rb

```diff
@@ -1,18 +1,20 @@
-class SaveAndSendRejectByDefaultFeedback
+class RejectByDefaultFeedback
   include ImpersonationAuditHelper
 
-  attr_reader :application_choice, :rejection_reason
+  attr_reader :application_choice, :rejection_reason, :structured_rejection_reasons
 
-  def initialize(actor:, application_choice:, rejection_reason:)
+  def initialize(actor:, application_choice:, rejection_reason: nil, structured_rejection_reasons: nil)
```

Comment:
That's a good idea! I can amend. I was following the pattern in another service, which can set this field https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/services/reject_application.rb#L11
I need to check if I also have to amend all the `where.not(structured_rejection_reasons: nil)` queries.

Reply:
If I made this change I would have to amend the queries that check for `nil` to also check for `{}`, so my personal preference would be to keep it `nil` and keep the queries simple but happy to make this change if you think otherwise.

Reply:
That's fair, since its not set to default in the db we can keep as is 👍